apps/condo/domains/meter/hooks/useTableColumns.tsx

```diff
@@ -149,54 +154,62 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
     }, [filters])
 }
 
+type ModalMessageKeys =
+    'InstallationDateMessage' | 'CommissioningDateMessage' | 'SealingDateMessage' | 'VerificationDateMessage' | 'NextVerificationDateMessage' | 'ControlReadingsDateMessage'
+
+const MODAL_MESSAGES: MessageSetMeta<ModalMessageKeys> = {
+    InstallationDateMessage: { id: 'pages.condo.meter.InstallationDate' },
+    CommissioningDateMessage: { id: 'pages.condo.meter.CommissioningDate' },
+    SealingDateMessage: { id: 'pages.condo.meter.SealingDate' },
+    VerificationDateMessage: { id: 'pages.condo.meter.VerificationDate' },
+    NextVerificationDateMessage: { id: 'pages.condo.meter.NextVerificationDate' },
+    ControlReadingsDateMessage: { id: 'pages.condo.meter.ControlReadingsDate' },
+}
+
+
 export const useMeterInfoModalTableColumns = () => {
     const intl = useIntl()
-    const InstallationDateMessage = intl.formatMessage({ id: 'pages.condo.meter.InstallationDate' })
```

Comment:
it's better to use an explicit translation (optional)
You want to change the common translation style